lib/presentation/widget/shared_space_document/shared_space_document_widget.dart

```diff
@@ -629,16 +629,11 @@ class _SharedSpaceDocumentWidgetState extends State<SharedSpaceDocumentWidget> {
               width: 24,
               height: 24,
               fit: BoxFit.fill),
-            onPressed: () =>  node.element.type == WorkGroupNodeType.FOLDER
-                ? sharedSpaceDocumentViewModel.openWorkGroupNodeContextMenu(
-                    context,
-                    node.element,
-                    _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder))
-                : sharedSpaceDocumentViewModel.openWorkGroupNodeContextMenu(
-                    context,
-                    node.element,
-                    _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
-                    footerAction: SharedSpaceOperationRole.deleteNodeSharedSpaceRoles.contains(_arguments?.sharedSpaceNode.sharedSpaceRole.name) ? _removeWorkGroupNodeAction([node.element]) : SizedBox.shrink())),
+            onPressed: () => sharedSpaceDocumentViewModel.openWorkGroupNodeContextMenu(
+              context,
+              node.element,
+              node.element.type == WorkGroupNodeType.FOLDER ? _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder) : _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
```

Comment:
What's the difference? Why this change helps to fix the bug?

Reply:
Should be break the line of code

Reply:
the footer was not added when it was folders before, I added it and I refactored a little bit so it's more understandable

Reply:
Oh, it's the footer, my eyes are so bad, LOL

Reply:
But it still needs to break the line IMO and fix the build failed too.

Reply:
Failed build, still by Certificate. Please ignore. fix the line
```
node.element.type == WorkGroupNodeType.FOLDER 
   ? _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder)
   : _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
```